crates/compilers/src/compilers/zksolc/settings.rs

```diff
@@ -9,12 +9,31 @@ use foundry_compilers_artifacts::{
 use semver::Version;
 use serde::{Deserialize, Serialize};
 use std::{
-    collections::BTreeSet,
+    collections::{BTreeSet, HashSet},
     fmt,
     path::{Path, PathBuf},
     str::FromStr,
 };
 
+/// `zksolc` warnings that can be suppressed.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Serialize, Deserialize)]
```

Comment:
It looks like I should seriously prioritize moving the entire interface of zksolc to another crate that doesn't depend on LLVM.
Hopefully, I will come up with a solution in the upcoming weeks.